holoviews/element/comparison.py

```diff
@@ -257,6 +262,11 @@ def compare_arrays(cls, arr1, arr2, msg='Arrays'):
                 raise cls.failureException(msg + str(e)[11:])
 
     @classmethod
+    def compare_dataframe(cls, df1, df2, msg='DataFrames'):
+        if not df1.equals(df2):
+            raise cls.failureException('%s are not equal' % msg)
+
```

Comment:
What happened before we added this? Surely we had some dataframes to compare in the notebook data tests?

Reply:
Nope I think we always avoided it somehow.